stable/datadog/templates/entrypoint-configmap.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "entrypoint.fullname" . }}
+  labels:
+    app: {{ template "entrypoint.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  entrypoint.sh: |
+    /bin/bash -eux
```

Comment:
This is a good idea, but the implementation is pretty hacky and will definitely break for a lot of setups.
Do you mind if we take it upstream? I think it fits in the agent, not in the deployment method. We have a few ideas for solving this (have a leader agent that would collect everything + events/cluster metadata, building a lightweight agent that would only collect events/metadata...). This issue will serve as reference: https://github.com/DataDog/dd-agent/issues/3113